### DISCUSSION TITLE
I get "Process EGLOT not running" although Eglot told me 'Connected' just before that.

### DISCUSSION THREAD

**Discussion by halloleo** [joaotavora/eglot#789] [Q&A] [answered]
Hi there

Firstly, thanks for Eglot. It is nice and small which is great!

I've been using Eglot for a while and it mostly worked alright :-), but now I get a strange issue when doing the following:

1. I load a Python file
2. Eglot tells me in the status line: *[eglot] Connected! Server 'EGLOT (MyProject/python-mode)' now managing 'python-mode' buffers in project 'MyProject'.*
3. I go to a symbol in the file and execute `eglot-rename`.
4. Then I get the following error message in the status line: *Process EGLOT (DataVisualisation/python-mode) not running*

Why's that and how can I fix this?

Many thanks for any pointers!

**Comment by joaotavora** [comment, marked as answer]
> Why's that

That's hard to say without since more information about how to reproduce your problem. Maybe you can open a big report issue and follow the template, which will guide you in providing such information.

But you do hint that it "worked" and "now" doesn't, which could also be used to pinpoint the problem. Again, without knowing what happened just before this "now", it's hard to analyze. Did you update Eglot recently? Your language server? Did you update same other aspect of your LSP setup?

If you provide some more information, we'll try to help you. Thanks

**Reply by halloleo** [reply]
Thanks @joaotavora for chiming in!

Here some first details:

* Server used:  **pyls**
* Emacs version:             **GNU Emacs 27.1** (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30) of 2021-09-07
* Operating system:          **Linux RHEL 8.3**
* Eglot version:             **20220109.2327**
* Eglot installation method: **`package.el`/`use-package` (use-package tried to load it but errored out, then I installed it manually via `list-packages`**
* Using Doom:                **No**

**Reply by halloleo** [reply]
Can you believe it: When I now tried to recreate the issue (to gather more detailed info) it was working now!

**Reply by halloleo** [reply]
But I will keep an eye on it and generate a proper traceback via `toggle-debug-on-error` next time I have the problem.
